### PR TITLE
better calculateIndices calculation

### PR DESCRIPTION
**Pull request by radiodario**
doing it better. doing it right.

### CODE REVIEW

File: src/server/utils/calculateIndices.js [hunk 1]
```diff
@@ -11,6 +11,9 @@ export default function calculateIndices(query, ES_INDEX) {
 
   let charF, charT;
   let i = 0;
+  // these are daily indices
+  // we need to minimise the lenght of the string
+  // and also minimising
```
Comment (katsuraku) [inline src/server/utils/calculateIndices.js]: last line of comments not needed?

File: test/utils/calculateIndices.spec.js [hunk 2]
```diff
@@ -40,41 +40,99 @@ describe('#calculateRealtimeIndices', () => {
...
+
+    const mFrom = moment(query.dateFrom);
+    const mTo = moment(query.dateTo);
+
+    const start = mFrom.clone();
+    const indices = [];
+    while(start <= mTo) {
+      indices.push(ES_INDEX + start.format("YYYY-MM-DD"))
+      start.add(1, 'day');
+    }
```
Comment (katsuraku) [inline test/utils/calculateIndices.spec.js]: Maybe this is too close to the implementation of the calculateIterateIndices? Would it be better if we just had a list of the dates that it should equal? Not sure, maybe it's not worth it and I'm being crazy.

**Comment by katsuraku** [comment]
algorithm foo :ok_hand:

File: test/utils/calculateIndices.spec.js (hunk 2 above)
Comment (jonathan-best) [inline test/utils/calculateIndices.spec.js]: I agree, could be worth splitting if it means we don't copy chunks of the same code into the unit tests

Comment (radiodario) [inline test/utils/calculateIndices.spec.js]: nah i kept it separate because i did TDD u guys - i wrote those before i wrote the function 💁

Comment (jonathan-best) [inline test/utils/calculateIndices.spec.js]: jvedtqd

Comment (jonathan-best) [inline test/utils/calculateIndices.spec.js]: jvedtqd

Comment (katsuraku) [inline test/utils/calculateIndices.spec.js]: TaDDah!

**Pull request merged**